app/src/components/common/layout/header/index.tsx

```diff
@@ -238,9 +198,16 @@ const HeaderContainer: React.FC = (props: any) => {
     </DropdownWrapper>
   )
 
+  const toggle = () => {
+    toggleRelay()
+    if (hasRouter) {
+      history.replace('/liquidity')
```

Comment:
Maybe better to just use `history.replace('/')` since it will redirect to /liquidity anyway and if we choose to change the routes in the future it won't cause any issues